src/Writers/WebWriter.swift

```diff
@@ -0,0 +1,19 @@
+
+import Foundation
+
+
+class WebWriter : Writer {
+    var configuration: Configuration
+        
+    init(configuration: Configuration) {
+        self.configuration = configuration
+    }
+    
+    func writeConstants(_ name: String, value: Any, level: Int) -> String {
```

Comment:
For these shared functions (b/t SwiftWriter, AndroidWriter, etc), it may be worthwhile to scaffold them in the superclass (Writer) and override it for customization.